imports/ui/Accordian.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/*
+  An accordian for the Statistics component.
+*/
+export default class Accordian extends Component{
+
+  getRandNum(){
+    return Math.floor((Math.random() * 1000000) + 0);
+  }
```

Comment:
The indentation is inconsistent with the rest. We might want to enforce consistent indentation in the linter.

Reply:
Will look into adding indentation to linter